From e2907eadaea661c2c2ff5d8cdd14f0b8f0172eab Mon Sep 17 00:00:00 2001
From: Michael Jeanson <mjeanson@efficios.com>
Date: Tue, 30 Jan 2018 16:30:36 -0500
Subject: [PATCH] Update kvm instrumentation for debian kernel 4.14.0-3

Signed-off-by: Michael Jeanson <mjeanson@efficios.com>
---
 instrumentation/events/lttng-module/kvm.h | 1 +
 1 file changed, 1 insertion(+)

--- a/instrumentation/events/lttng-module/kvm.h
+++ b/instrumentation/events/lttng-module/kvm.h
@@ -86,6 +86,7 @@
 
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(4,15,0) \
 	|| LTTNG_KERNEL_RANGE(4,14,14, 4,15,0) \
+	|| LTTNG_DEBIAN_KERNEL_RANGE(4,14,13,0,1,0, 4,15,0,0,0,0) \
 	|| LTTNG_KERNEL_RANGE(4,9,77, 4,10,0) \
 	|| LTTNG_KERNEL_RANGE(4,4,112, 4,5,0) \
 	|| LTTNG_KERNEL_RANGE(3,16,52, 3,17,0) \
